src/helpers/fetchLocation.js

```diff
@@ -0,0 +1,13 @@
+const myKey = 'at_vEhT0ZnWw5eYL3XlfArBnWieonjhS';
```

Comment:
Please move it to environment variables. As we discussed, you never should store any secrets in your code base.
Here is how you can do this https://vitejs.dev/guide/env-and-mode.html

Reply:
done

Reply:
Man, when you say done - push your changes please. Because I cannot see what exactly has been done.
In this particular example I wanted to grab your env variable to add it to our GitHub Pages.